src/app/Vault/index.tsx

```diff
@@ -52,6 +52,25 @@ const tabExplanations: { [key in TabNames]: ReactNode } = {
 export const Vaulted = () => {
   const [currentTab, setCurrentTab] = useState<TabNames>(TabNames.VAULTED);
   const { state } = useContext(StateContext);
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowWidth(window.innerWidth);
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (windowWidth <= 640 && currentTab === TabNames.VAULTED) {
```

Comment:
abstract out the magic number into it's own variable in caps to make this easy to understand / configure.

so something like

```
const MOBILE_WIDTH = 640;

useEffect(...
```

Reply:
Also, you could consider using a `useIsMobile` hook to abstract this out and make this more declarative

Reply:
added MOBILE_WIDTH and migrated logic into useIsMobile hook